campbot/checkers.py

```diff
@@ -461,7 +463,7 @@ class QualityTest:
 
     def __init__(self):
         self.name = "Changement du champ qualité"
-        self.fail_marker = "🧹"
+        self.fail_marker = ""
```

Comment:
Is it a fat finger, or you removed it on purpose?

Reply:
It is a fat finger! 
On my screen, I see a strange empty square, not the broom...

Reply:
It means that your default font is quite old :) Is it the same on your the forum on reports ?

Reply:
Ok ,then juste put it back, and it's ok 

```suggestion
        self.fail_marker = "🧹"
```